### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by tsregit**
## Information
-Initial commit with content layout
-Add css and Images.
-Add animation on div.
## Screenshots
image

### CODE REVIEW

File: scss/titles.scss [hunk 1]
```diff
@@ -0,0 +1,15 @@
+.title-image {
+  width: 90px;
+  height: 70px;
+}
+
+.title-decoration {
+  margin-left: -100px;
```
Comment (arconejeros) [inline scss/titles.scss]: avoid using negative margins

File: scss/titles.scss [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  margin-top: 80px;
+  display: flex;
+  align-items: center;
+}
+
+.title-text {
+  margin-left: -31px;
```
Comment (arconejeros) [inline scss/titles.scss]: Same here

File: index.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  <section>
+    <div class="title-decoration">
+      <img class="title-image" src="triangle.svg" alt="Front-End Tools"/>
+      <p class="title-text">Front-End Tools</p>
+    </div>
+    <div class="container-tech">
+      <div class="tech">
+          <img  class="tech-img" src="react_icon.svg" alt="ReactJS">
+          <p class="text-big">ReactJS</p>
+          <br>
```
Comment (arconejeros) [inline index.html]: `<br>` tags shouldn't be used to make gaps between elements, instead adjust the margin styled by CSS

**Comment by gon-escalante** [comment]
add a screenshot in the PR as a reference, it helps a lot

File: scss/tech.scss [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+.tech {
+  position: relative;
+  justify-content: space-between;
+  flex-direction: column;
+  align-items: center;
+  width: 200px;
+  background-color: $tech-green;
+  margin: 15px;
+  padding: 20px;
+  border-radius: 5px 5px 5px 5px;
```
Comment (gon-escalante) [inline scss/tech.scss]: `border-radius: 5px;`
it applies to all 4 borders by default

File: scss/tech.scss [hunk 5]
```diff
@@ -0,0 +1,84 @@
...
+  -moz-border-radius: 5px 5px 5px 5px;
```
Comment (gon-escalante) [inline scss/tech.scss]: same

File: scss/tech.scss [hunk 6]
```diff
@@ -0,0 +1,84 @@
...
+  -webkit-border-radius: 5px 5px 5px 5px;
```
Comment (gon-escalante) [inline scss/tech.scss]: same

**Review by gon-escalante — CHANGES_REQUESTED** [review]

**Review by gon-escalante — APPROVED** [review]

**Pull request merged**